changelog.d/5142.bugfix

```diff
@@ -0,0 +1 @@
+Aligns use case identifying with iOS implementation
```

Comment:
Maybe prefix the content with `Analytics: ` else it can be a bit confusing.
Or create a new extension for towncrier `.analytics` which will end to a dedicated section in the Changelog?

Reply:
a new section could be handy :thinking: I think for now the prefix might be enough~

Reply:
3f3cac0489342ea0fdcd90fe80237bd711bfe77c